server/controllers/clubProfileController.js

```diff
@@ -1,8 +1,9 @@
+const db = require('../models/')
+
 module.exports = {
     getAll: function(req, res) {
-        // TODO
-        // support pagination with req.query 
-        // support shuffle (i.e. randomize entry order)?? 
+	 var query = db.find({}).select({name: 1, description: 1, imageUrl: 1, tags: 1, memberRange: 1, acceptingMembers: 1, applicationRequired: 1});
+	 res.send(query);
```

Comment:
awesome! Could you refer to Tanner's PR #78 to see how we generally format the query and the responses in each controller function, and modify based on that? (e.g. use `.then` connectors and also catch errors).